code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -9,11 +9,14 @@
 				new /obj/item/clothing/mask/gas/voice(src)
 				new /obj/item/weapon/card/id/syndicate(src)
 				new /obj/item/clothing/shoes/syndigaloshes(src)
+				new /obj/item/weapon/storage/box/syndie_kit/surveillance(src)
+				new /obj/item/weapon/dnascrambler(src)
 
 			if("stealth")
 				new /obj/item/weapon/gun/energy/crossbow(src)
 				new /obj/item/weapon/pen/paralysis(src)
 				new /obj/item/device/chameleon(src)
+				new /obj/item/weapon/soap/syndie(src)
```

Comment:
I don't like all these random unlisted buffs to the random kits. For one thing, it doesn't even seem atomic.

And the additions seem random. Why syndie soap for the stealth bundle? Did the spai bundle feel like it needed a 10TC buff?